Controllers/controller.py

```diff
@@ -106,3 +108,21 @@ def open_file_dialog(self):
             url = file_dialog.selectedUrls()[0]
             self._media_player.setSource(url)
             self._media_player.play()
+            
+    @Slot()
+    def play_video(self):
+        '''Command a video to change the state to play and pause when clicked'''
+        if self._media_player.playbackState() == QMediaPlayer.PlayingState:
+            self._media_player.pause()
+        else:
+            self._media_player.play()
+        self.toggle_play_pause_icon()
+
+    @Slot()
+    def toggle_play_pause_icon(self):
+        '''Changes the video state and changes the icon of play and pause'''
```

Comment:
Replace the single quotations with double quotations. """

This function does not change the video state, it only changes the icon of the button. Instead, this would be better.
`"""Toggles the icon of the play/pause button.`